models/issue_comment.go

```diff
@@ -1163,7 +1163,7 @@ func deleteComment(e db.Engine, comment *Comment) error {
 	}
 
 	if comment.Type == CommentTypeComment {
-		if _, err := e.Exec("UPDATE `issue` SET num_comments = num_comments - 1 WHERE id = ?", comment.IssueID); err != nil {
+		if _, err := e.ID(comment.IssueID).Decr("num_comments").Update(new(Issue)); err != nil {
```

Comment:
Hmm... `xorm.io/xorm/internal/statements/statement.go`:

```go
// Decr Generate  "Update ... Set column = column - arg" statement
func (statement *Statement) Decr(column string, arg ...interface{}) *Statement {
	if len(arg) > 0 {
		statement.DecrColumns.Add(column, arg[0])
	} else {
		statement.DecrColumns.Add(column, 1)
	}
	return statement
}
```

`xorm.io/xorm/session_update.go`:

```go
  ...
	// for update action to like "column = column - ?"
	decColumns := session.statement.DecrColumns
	for _, expr := range decColumns {
		colNames = append(colNames, session.engine.Quote(expr.ColName)+" = "+session.engine.Quote(expr.ColName)+" - ?")
		args = append(args, expr.Arg)
	}
```

So I'm not sure that this is going to do anything different.

Reply:
no it should do the same - just coding style fix